app/routes/user/password.tsx

```diff
@@ -0,0 +1,235 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+import type { DataFunctionArgs } from '@remix-run/node'
+import { Link, useFetcher, useRouteError } from '@remix-run/react'
+import { Button, Icon, Label, TextInput } from '@trussworks/react-uswds'
+import { useState } from 'react'
+
+import { updatePassword } from './password.server'
+import Spinner from '~/components/Spinner'
+import { getFormDataString } from '~/lib/utils'
+import { useUserIdp } from '~/root'
+
+export const handle = {
+  breadcrumb: 'Password',
```

Comment:
This should match the link text in the nav bar:

```suggestion
  breadcrumb: 'Reset Password',
```